rocm-validation-suite/PKGBUILD

```diff
@@ -2,26 +2,22 @@
 # Contributor: codyfish <fe27 at gmx dot net>
 # Contributor: sofiageo <george at sofianos dot dev>
 pkgname=rocm-validation-suite
-pkgver=5.0.2
+pkgver=5.1.0
 pkgrel=1
-pkgdesc="Tool for detecting and troubleshooting common problems affecting AMD
-GPUs"
+pkgdesc="Tool for detecting and troubleshooting common problems affecting AMD GPUs"
 arch=('x86_64')
 url="https://github.com/ROCm-Developer-Tools/ROCmValidationSuite"
 license=('MIT')
 depends=('pciutils' 'doxygen' 'rocblas' 'rocm-smi-lib64' 'git' 'libpciaccess')
 makedepends=('cmake')
 options=(!staticlibs strip !lto)
 source=("$pkgname-$pkgver.tar.gz::https://github.com/ROCm-Developer-Tools/ROCmValidationSuite/archive/rocm-$pkgver.tar.gz"
-        "gtest.patch::https://patch-diff.githubusercontent.com/raw/ROCm-Developer-Tools/ROCmValidationSuite/pull/523.patch"
```

Comment:
Should this patch be replaced by this one: https://github.com/ROCm-Developer-Tools/ROCmValidationSuite/pull/533 ?
I'm surprised this built for you.

Reply:
I can confirm it builds fine on my computer, though I did not use a chroot.  I'll have to setup chroot on my computer first though because `paru -Ui --chroot` does not work for me.

Reply:
thx for confirming

Reply:
> Remove a patch which has been included in this version.

Can you point to the commit for this? Dig some digging and in 5.1.0 this is still 1.10.0
https://github.com/ROCm-Developer-Tools/ROCmValidationSuite/blob/rocm-5.1.0/CMakeGtestDownload.cmake#L33